### PR TITLE
refactor: - some error throw fix

### PR DESCRIPTION
**Pull request by nicolasburtey**
- update docker-compose
- fix some test

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ services:
     - lnd2
     - lnd-outside-1
     - lnd-outside-2
+    - otel-agent
```
Comment (bodymindarts) [inline docker-compose.yml:14]: I don't think `otel-agent` is a dependency of integration tests is it?

**Review by bodymindarts — APPROVED** [review]
Approving but think `otel-agent` was mistakenly added.

File: docker-compose.yml (hunk 1 above)
Comment (nicolasburtey) [reply docker-compose.yml:14]: it was added on purpose to get telemetry when doing `make integration`

Comment (nicolasburtey) [reply docker-compose.yml:14]: won't do anything unless the env var are being set. but maybe that would be useful to get this in the CI server to see the difference between the tests over time. like if we introduce regression we would could it?

Comment (nicolasburtey) [reply docker-compose.yml:14]: like this trace is interesting/intriguing: https://ui.honeycomb.io/galoy/datasets/nicolas-dev/result/oi9y9hjhAqF?tab=raw the consistent 3s (timeout?) could highlight a bug

**Pull request merged**